scripts/release.sh

```diff
@@ -30,7 +30,7 @@ fi
 
 # Fix until we can update this script to follow semantic versioning
 if [[ $WORKFLOW_NAME == amr ]]; then
-    TAG="amr-v1.3.1"
+    TAG="amr-v1.3.2-beta.1"
```

Comment:
To be honest, I think we can remove this code. If we merge this into `main` and release it, it should go to `amr-v1.3.2`.